caiman/motion_correction.py

```diff
@@ -3076,6 +3076,9 @@ def tile_and_correct_wrapper(params):
     extension = extension.lower()
     shift_info = []
 
+    print("In tile and correct wrapper about to call load with subidxs")
```

Comment:
These messages are noisy when run in the notebook

Reply:
Oops. Cutting errant debugging msg.